tests/CustomerAndServiceNode.test.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
```

Comment:
This is an awesome integration test. WIll travis ci be able to run this? I guess we'll find out soon. I'll set up travis as soon as we have merged this PR

Reply:
Yes, I enabled travis for my fork and this test passed 👍